golem_messages/message.py

```diff
@@ -1358,6 +1358,39 @@ def deserialize_slot(self, key, value):
         return deserialize_task_to_compute(key, value)
 
 
+class FileTransferToken(Message):
+    TYPE = CONCENT_MSG_BASE + 5
+
+    __slots__ = [
+        'token_expiration_deadline',
+        'storage_cluster_address',
+        'authorized_client_public_key',
+        'files',
+        'operation',
+        'file_sizes',
+        'md5_hashes',
+    ] + Message.__slots__
+
+    def __init__(

```

Comment:
Since all params are defaulted to `None` we can omit definition of `__init__` altogether like in `ForceReportComputedTask`?